C/c4Collection.cc

```diff
@@ -24,42 +24,39 @@ using namespace litecore;
 
 
 C4Collection::C4Collection(C4Database *db, C4CollectionSpec spec)
-:_database(db)
-,_scope(spec.scope)
-,_name(spec.name)
-{ }
+        : _database(db), _scope(spec.scope), _name(spec.name) {}
 
 
 void C4Collection::failClosed() const {
-   C4Error::raise(LiteCoreDomain, kC4ErrorNotOpen,
-                  "Invalid collection: either deleted, or db closed");
+    C4Error::raise(LiteCoreDomain, kC4ErrorNotOpen,
+                   "Invalid collection: either deleted, or db closed");
 }
 
 
-C4Database* C4Collection::getDatabase() {
+C4Database *C4Collection::getDatabase() {
     if (_usuallyFalse(!_database))
         failClosed();
     return _database;
 }
 
 
-const C4Database* C4Collection::getDatabase() const {
+const C4Database *C4Collection::getDatabase() const {
     if (_usuallyFalse(!_database))
         failClosed();
     return _database;
 }
 
 
-C4Document* C4Collection::documentContainingValue(FLValue value) noexcept {
+C4Document *C4Collection::documentContainingValue(FLValue value) noexcept {
     auto doc = VectorDocumentFactory::documentContaining(value);
     if (!doc)
         doc = TreeDocumentFactory::documentContaining(value);
     return doc;
 }
 
 
-Retained<C4Query> C4Collection::newQuery(C4QueryLanguage language, slice expr,int *errPos) const {
+Retained<C4Query> C4Collection::newQuery(C4QueryLanguage language, slice expr, int *errPos) const {
     if (_usuallyFalse(!_database))
         failClosed();
-    return C4Query::newQuery(const_cast<C4Collection*>(this), language, expr, errPos);
+    return C4Query::newQuery(const_cast<C4Collection *>(this), language, expr, errPos);
```

Comment:
Putting a space before the `*` doesn't make sense here; `C4Collection*` is a type.

The reason for the spacing comes from pointer variable declarations like `char *foo` where it's supposed to highlight that `*foo` is a `char`. It also promotes writing multiple declarations correctly, like `char *foo, *bar` instead of `char* foo, bar` (where bar gets the wrong type.)

There's no reason to put a space before a `*` (or `&`) in another context; it's not clarifying anything then.

Reply:
Fixed with move to Clang-Format